src/pages/home/sidebar/ChatLinkRow.js

```diff
@@ -77,6 +83,7 @@ const ChatLinkRow = ({
                         styles.alignItemsCenter,
                     ]}
                 >
+                    {/* FOR SINGLE USER DM WITH ICON */}
```

Comment:
NAB, but I wouldn't use caps for comments, other view comments are normal cased.

Reply:
Alright. Fix coming up in my next commit.